test/sav/sav_compare.c

```diff
@@ -356,6 +356,10 @@ static int is_exception_image_grid(int global_offset)
     if ((v1 == 734 && is_between(v2, 743, 770)) || (v2 == 734 && is_between(v1, 743, 770))) {
         return 1;
     }
+    // road to granary: in julius the roads lead into the granary
+    if (both_between(v1, v2, 591, 657)) {
```

Comment:
Please split it up into two checks: 591-639 for the paved roads, and 640-657 for the dirt roads. I don't quite like the fact that we're now basically not checking any road graphics at all, so at least make a difference between dirt/paved roads 😉

Reply:
Is there a way to checo from a savefile if a granary is adjacent to the tile?

Reply:
Yes, you would check the same way as in the game but then in the raw data. It's quite cumbersome: first you have to look up the building id for a grid offset in part `"building_grid"`, then you have to look into part `"buildings"` for that building (entries of 128 bytes) whether the type as short at index 10 (0-based) is equal to 71 (granary).

Reply:
Thanks for the info, I'll try to implement it.

Reply:
Before I commit and push the final changes, the code is now only excempting from tests only the road tiles that are adjacent to granaries, so the normal roads are tested again. Do you still want a separation between the paved roads and dirt roads?

Reply:
If we only check those roads next to the granary, it's OK to skip the paved/dirt road distinction.